### PR TITLE
Backpack items & home space fix

### PR DESCRIPTION
**Pull request by blukai**

### CODE REVIEW

**Comment by blukai** [comment]
image

**Comment by howardchung** [comment]
Think we should use a separate column for backpack?

File: src/components/Home/Buttons.jsx [hunk 1]
```diff
@@ -13,12 +13,8 @@ export default () => (
       <FlatButton
         label={
           <span className={styles.label}>
```
Comment (howardchung) [inline src/components/Home/Buttons.jsx]: Could we use a single span separated by space?

**Comment by blukai** [comment]
> Think we should use a separate column for backpack?

In this case, items 1 - 6 will be on two lines 😆 
I think I can add backpack icon to make it recognisable

**Comment by howardchung** [comment]
Check a lone druid game to see if it works well there (three rows of items?)

**Comment by blukai** [comment]
image

**Pull request merged**